specification/storage/resource-manager/Microsoft.Storage/stable/2023-01-01/storage.json

```diff
@@ -3101,7 +3101,7 @@
         "allowBlobPublicAccess": {
           "type": "boolean",
           "x-ms-client-name": "AllowBlobPublicAccess",
-          "description": "Allow or disallow public access to all blobs or containers in the storage account. The default interpretation is true for this property."
+          "description": "Allow or disallow public access to all blobs or containers in the storage account. The default interpretation is false for this property."
```

Comment:
This looks is breaking change. Please ask for breaking change review.

Reply:
breaking change review has happened, updated details in email thread. please let me know if further steps are needed.